### ISSUE TITLE
Migrate to using Web API instead of webhooks

### ISSUE THREAD

**Issue by patcon** [ossobv/slackbridge#10]
The Slack Web API _does_ allow apps to write messages as_user, but it's a little tricky:
https://codexample.org/questions/417349/is-it-possible-to-post-to-chat-postmessage-as-any-user-in-a-slack-team.c#answer-880240

Using Web API would allow:

- Sending reactions across bridge (would require asking each "reacting" user to auth with app, which is no small feat)
- Sending edits across bridge
- Sending "user is typing" messages (using [`typing` in RTM api](https://api.slack.com/rtm#typing_indicators))

I agree that this is probably not priority, and so would require someone else to do the work. I would also remove the simplicity of the current approach... I would understand if you think this should be tackled by a separate project :)

**Comment by wdoekes** [comment]
Yes, I did another bot for a different purpose and did see the possibility to do the above, but getting it all right is rather cumbersome. For now, I'd rather keep things simple. My first priority would be to make the setup (configuration and token generation) simpler.

**Comment by patcon** [comment]
Cool! I'll leave this open for posterity, but feel free to close if that's your triage pref
